src/main/java/uk/gov/hmcts/reform/sscs/ccd/client/CcdClient.java

```diff
@@ -150,4 +87,28 @@ private CaseDetails submitEventForCaseworker(IdamTokens idamTokens, String caseI
                         .putAll(searchCriteria)
                         .build());
     }
+
+    public CaseDetails readForCaseworker(IdamTokens idamTokens, Long caseId) {
+        return coreCaseDataApi.readForCaseWorker(
+                idamTokens.getIdamOauth2Token(),
+                idamTokens.getServiceAuthorization(),
+                idamTokens.getUserId(),
+                ccdRequestDetails.getJurisdictionId(),
+                ccdRequestDetails.getCaseTypeId(),
+                caseId.toString()
+        );
+    }
+
+    public EventRequestData getEventRequestData(IdamTokens idamTokens, String eventId) {
+        log.info("getEventRequestData...");
```

Comment:
this is not very helpful. I think this line has been copied and pasted many times but it doesn't say much, does it? We could clean it up.

Reply:
Not used so I can delete